data/intro missions.txt

```diff
@@ -451,3 +451,139 @@ conversation "end of intro missions"
 	label next
 	`	There's an awkward pause. You're unsure of how to respond. Then he says, "Thing is, this here planet is one of the most exclusive retirement communities in the galaxy. Plenty of the locals are probably worth more than a billion credits. But they're mostly financial or business types, or old money. I'm rich enough to be here, but I just hope they're willing to socialize with someone who made his fortune flying a ship."`
 	`	He hands you a credit chip worth <payment>, and says, "Best of luck to you, Captain. Don't hesitate to look me up if you're ever in the area."`
+
+
+
+mission "Intro: Follow Up - Hai Before Main Plot"
```

Comment:
this is an established naming convention.  for example:  `mission "Smuggler's Den: Follow Up"` the previous missions in this case are called `Intro` so the consistent syntax is to append a `: Follow Up`

Possibly the text I have after a ` - ` should actually go into square brackets to denote the various branches.  This might be the more consistent thing to do.